DataManagerPlugin.cs

```diff
@@ -1,13 +1,167 @@
-using BepInEx;
+using Bep = BepInEx;
+using CG = System.Collections.Generic;
+using HL = HarmonyLib;
+using IO = System.IO;
+using Json = Newtonsoft.Json;
+using UE = UnityEngine;
 
 namespace Silksong.DataManager;
 
-[BepInAutoPlugin(id: "org.silksong-modding.datamanager")]
-public partial class DataManagerPlugin : BaseUnityPlugin
+[Bep.BepInAutoPlugin(id: "org.silksong-modding.datamanager")]
+public partial class DataManagerPlugin : Bep.BaseUnityPlugin
 {
-    private void Awake()
+    internal static DataManagerPlugin? Instance;
```

Comment:
1. This should be a property so it cannot be reassigned outside the class (not that we would, but let's be principled)
2. Rather than flagging this as nullable and null forgiving at every usage point, I'd rather lie to the null checker once here and say it can't be null